scripts/docker/Dockerfile

```diff
@@ -1,6 +1,5 @@
 # Multi-stage builder to avoid polluting users environment with wrong
-# architecture binaries.  Since this binary is used in an alpine container,
-# we're explicitly compiling for 'linux/amd64'
+# architecture binaries.
```

Comment:
I removed this comment about `linux/amd64` as I was able to successfully get this to work on `linux/arm64` on my Mac as well (but not `Dockerfile.ui` due to problems with the `debian` distro).